### PR TITLE
[EDU-5636] Update release notes

### PR DESCRIPTION
**Pull request by guiafonso-ol**
### Changes

Updates release notes.

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/release-notes/release-notes.mdx [hunk 1]
```diff
@@ -9,6 +9,56 @@ permalink: /documentation/products/release-notes/
...
+
+To learn more about Azion Console, check out the [documentation](/en/documentation/products/guides/getting-to-know-azion-console/).
+
+---
+
+## November 8, 2024
+
+### Azion CLI
+
+**Version 2.3.0**
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/main-menu/release-notes/release-notes.mdx:28]: Azion CLI 2.3.1

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx [hunk 2]
```diff
@@ -9,6 +9,56 @@ permalink: /documentacao/produtos/release-notes/
...
+
+Para saber mais sobre o Azion Console, consulte a [documentação](/pt-br/documentacao/produtos/guias/conhecendo-o-azion-console/).
+
+---
+
+## 8 de novembro, 2024
+
+### Azion CLI
+
+**Versão 2.3.0**
```
Comment (GabrielAzion) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:28]: Versão 2.3.1

**Review by isidrohernandezazion — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/release-notes/release-notes.mdx (hunk 1 above)
Comment (guiafonso-ol) [inline src/content/docs/en/pages/main-menu/release-notes/release-notes.mdx:28]:
```suggestion
**Version 2.3.1**
```

File: src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx (hunk 2 above)
Comment (guiafonso-ol) [inline src/content/docs/pt-br/pages/menu-principal/release-notes/release-notes.mdx:28]:
```suggestion
**Versão 2.3.1**
```

**Pull request merged**